$NetBSD: patch-ba,v 1.3 2006/09/27 15:28:50 rillig Exp $

When checking for gcc>=3, the correct operator is -ge, not =.

--- configure.orig	2005-07-08 09:55:43.000000000 +0200
+++ configure	2006-09-27 17:17:58.000000000 +0200
@@ -3198,7 +3198,7 @@ echo $ECHO_N "checking the GNU gcc compi
    _gcc_major=`echo $_gcc_version | $AWK -F. '{ print \$1 }'`
    _gcc_longver=`echo $_gcc_version | $AWK -F. '{ print \$1*10000+\$2*100+\$3 }'`
 
-   if test "$_gcc_major" = "3"; then
+   if test "$_gcc_major" -ge "3"; then
       USE_GCC3="TRUE"
       if test "$_gcc_longver"  -eq "030203"; then
 		 if test "$ENABLE_SYMBOLS" = "SMALL"; then
